toolkit/scripts/toolchain/container/toolchain_build_temp_tools.sh

```diff
@@ -51,7 +51,7 @@ case $(uname -m) in
     sed -e '/m64=/s/lib64/lib/' -i.orig gcc/config/i386/t-linux64
   ;;
   aarch64)
-    GCC_CONFIG_WITH_ARCH="armv8.1-a"
+    GCC_CONFIG_WITH_ARCH=""
```

Comment:
Is setting `--with-arch=""` equivalent to not supplying `--with-arch` at all in the configure script? I'm concerned that specifying "" is actually different than what the default value would be (and if true, which one is the correct one we want?)

Unfortunately https://gcc.gnu.org/install/configure.html is light on information about this...

Reply:
Good question. It appears to be the same, although I've made a change so this is not set at all for arm64

Reply:
I tested the build both ways and shared some outputs below.

Configured with: ../configure … --with-arch= …
COLLECT_GCC_OPTIONS='-v' '-mlittle-endian' '-mabi=lp64' '-dumpdir' 'a-'

Configured with: ../configure … (do not call --with-arch at all)
COLLECT_GCC_OPTIONS='-v' '-mlittle-endian' '-mabi=lp64' '-dumpdir' 'a-'

Configured with: ../configure … --with-arch=armv8.1-a …
COLLECT_GCC_OPTIONS='-v' '-march=armv8.1-a' '-mlittle-endian' '-mabi=lp64' '-dumpdir' 'a-'